go.mod

```diff
@@ -1,4 +1,4 @@
-module decred.org/dcrwallet/v3
+module decred.org/dcrwallet/v4
 
 go 1.17
```

Comment:
i asked the dev room if anyone was opposed to increasing the minimum go version to 1.19 when we move to v4, and didn't hear any opposition. so let's do that as well.  it'll give us access to some nice new things like generics for example.